src/commands/close-project.js

```diff
@@ -0,0 +1,50 @@
+const flow = require('lodash.flow');
+const { URL } = require('url');
+
+const github = require("../lib/github");
+const { withToken } = require('../lib/helpers/yargs/options');
+
+/**
+ * yargs builder function.
+ *
+ * @param {import('yargs').Yargs} yargs - Instance of yargs
+ */
+const builder = yargs => {
+	const baseOptions = flow([withToken]);
+
+	return baseOptions(yargs)
+		.positional('path', {
+            describe: 'URL path to project',
+            type: 'string'
+		});
+};
+
+/**
+ * Close a project.
```

Comment:
I'm confused... the commit message says "List all open projects for an org" :thinking: